HousingRegisterApi/build.cmd

```diff
@@ -1,2 +1,2 @@
 dotnet restore
-dotnet lambda package --configuration release --framework net6.0 --output-package bin/release/net6.0/housing-register-api.zip
+dotnet lambda package --configuration release --framework net8.0 --output-package bin/release/net/housing-register-api.zip
```

Comment:
is this path correct?

Reply:
it wasn't 🤦 
addressed in [953cee1](https://github.com/LBHackney-IT/lbh-housing-api/pull/240/commits/953cee1f8c87a3f6f2b274cf962ef90d3edfe0cb)